10-unique_list.py

```diff
@@ -0,0 +1,9 @@
+def unique():
+    unique_list=list(input('Please enter list items: '))
+    #unique_list=[1, 2, 3, 3, 3, 3, 4, 5, 5]
+    new_list=[]
+    for i in unique_list:
+        if i not in new_list:
+            new_list.append(i)
+    print(new_list)
+unique()
```

Comment:
- kodlarınız çalışıyor. tebrikler
- açıklama eklerseniz güzel olur.
bu soru için aşağıda verdiğim kodlar daha basit ve hızlı bir çözüm yöntemi sunuyor. incelemenizi tavsiye ederim.

def unique_list(users_list):
    """filters unique elements o a list"""
    users_list = set(users_list)
    return list(users_list)

print(unique_list([9, 1, 2, 3, 3, 3, 3, 4, 5, 5]))